cloudfoundry/structures_app.go

```diff
@@ -333,6 +343,7 @@ func AppDeployV3ToResourceData(d *schema.ResourceData, appDeploy v3appdeployers.
 	_ = d.Set("stack", appDeploy.App.StackName)
 	if bpkg := appDeploy.App.LifecycleBuildpacks; len(bpkg) > 0 {
 		_ = d.Set("buildpack", bpkg[0])
+		_ = d.Set("buildpacks", bpkg)
```

Comment:
I just realized with this line subsequent calls to `terraform apply` will not be consistent because `buildpack` is set in terraform state but not in `.tf` files. I think you need to add a separate `else` block for `buildpacks` here.

Reply:
Yes I can confirm that... I seperated buildpacks and buildpack and just write one, if it is set in resource data. Therefore I had to prevent buildpacks to be computed, so it keeps writing only buildpack, if nothing is set.